### ISSUE TITLE
Ensure that flutter apps can be navigated with a physical hardware switch

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#11004]

**Comment by goderbauer** [comment]
Switch control uses slightly different semantics then TalkBack, needs to be tested separately.

Can use a bluetooth keyboard to simulate this on Android for TalkBack.

**Comment by goderbauer** [comment]
I ordered a Bluetooth keyboard to test this.

**Comment by goderbauer** [comment]
Quick and basic test on Android: Seems to work fine.
On iOS: It's only doing line scanning and I cannot get it to do item scanning.

**Comment by jeowen** [comment]
Writing on behalf of developers working for the US Department of Veterans Affairs.  We are hoping to develop federal applications that serve hundreds of thousands of U.S. Veterans and service members, but all of our applications require adherence to strict accessibility standards. Hoping this issue is something that can be addressed so that we, and the many Veterans who require hardware to utilize mobile applications, can use Flutter.

**Comment by gmoraja** [comment]
Here is more information about the issue.

Mobile apps do not seem to work with an external keyboard connected via Bluetooth: on the keyboard, the tab or arrow keys fail to change the point of focus. This is noted on both iOS and Android, with or without Assistive Technology (Voiceover for iOS and TalkBack for Android).

**Comment by goderbauer** [comment]
@gmoraja What you describe sounds like you want to navigate your app with the keyboard, which is different from whats described in this issue. This issue is about the assistive technology Switch Access (https://support.google.com/accessibility/android/answer/6122836?hl=en). You can configure a Bluetooth keyboard to act like a switch for switch access, but that would generally not use tab or arrow keys to change focus. 

Keyboard navigation was covered in https://github.com/flutter/flutter/issues/13264, which is closed now. Is there something that's missing for your use case?

**Comment by gmoraja** [comment]
@goderbauer thanks for the clarification. Yes, the main issue for us is to enable tab navigation with an external keyboard. In the layout, we need to be able to define the focus order for each of the focusable elements, with or without Assistive Technology (Voiceover/TalkBack).

We noticed issue #13264 has been closed but we're still experiencing the problem.

**Comment by goderbauer** [comment]
Can you file a new issue, ideally with an example of where it is currently not working so we can investigate it? Please tag me (@goderbauer) in the new issue so I'll see it.

Also, I was just informed that it's expected to not work on iOS as we don't have keyboard support there yet, but it should work on Android.

**Comment by gmoraja** [comment]
@goderbauer I just opened issue #47479. Thanks for the heads-up on iOS.

**Comment by chunhtai** [comment]
This is an old issue and I think both switch access on Android and switch control on iOS are working for flutter app. Any particular issues were and should be filed separately. Closing this one
